zephyr/esp32s2/CMakeLists.txt

```diff
@@ -79,10 +99,10 @@ if(CONFIG_SOC_ESP32S2)
 
   zephyr_sources(
     ../../components/soc/esp32s2/gpio_periph.c
-	../../components/esp_hw_support/port/esp32s2/rtc_clk.c
-	../../components/esp_hw_support/port/esp32s2/rtc_init.c
-	../../components/esp_hw_support/port/esp32s2/rtc_time.c
-	../../components/esp_hw_support/port/esp32s2/rtc_clk_init.c
+    ../../components/esp_hw_support/port/esp32s2/rtc_clk.c
```

Comment:
Is this indentation wrong?

Reply:
Yes, it was lacking 2 spaces. I used the same PR to adjust it.